packages/help-center/src/components/help-center.tsx

```diff
@@ -49,12 +53,24 @@ const HelpCenter: React.FC< Container > = ( {
 	const { hasActiveChats, isEligibleForChat } = useChatStatus();
 	const { isMessagingScriptLoaded } = useLoadZendeskMessaging(
 		'zendesk_support_chat_key',
-		( isHelpCenterShown && isEligibleForChat ) || hasActiveChats,
-		isEligibleForChat || hasActiveChats
+		isHelpCenterShown && isEligibleForChat,
+		isEligibleForChat
 	);
 
 	useZendeskMessagingBindings( HELP_CENTER_STORE, hasActiveChats, isMessagingScriptLoaded );
 
+	const { initSmooch, destroy } = useSmooch();
+	const ref = useRef( null );
+
+	useEffect( () => {
+		if ( isMessagingScriptLoaded && ref?.current ) {
```

Comment:
Ref will never be null.

```suggestion
		if ( isMessagingScriptLoaded && ref.current ) {
```